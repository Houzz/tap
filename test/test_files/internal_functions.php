<?php

function testInernalFuncs() {
    // $a = array(1, 2);
    // __tap_check($a, "vector<int>");
    $b = abs(1);
    $c = abs("str");    // wrong
}

