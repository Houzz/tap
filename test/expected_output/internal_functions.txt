7 E_FUNCTION_ARGUMENT
